asapdiscovery-alchemy/asapdiscovery/alchemy/predict.py

```diff
@@ -989,3 +989,76 @@ def download_cdd_data(protocol_name: str) -> pd.DataFrame:
     )
 
     return formatted_data
+
+
+def clean_result_network(network, console=None):
+    """
+    Cleans an incoming result network JSON file from some issues that might occur. Current procedures:
+    - removes edges that have DG==0.0, this happens when there is inconsistent stereo annotation in input ligands
+    such that after stereo enumeration there are duplicate ligands.
+    - cleans imbalanced complex/solvent legs, e.g. when some have failed or when stereo expansion was done unintionally
```

Comment:
This can also happen due to https://github.com/OpenFreeEnergy/openfe/issues/882, we should make a note of that.